scylla_elasticsearch_integration/insert_data_into_scylla+elastic.py

```diff
@@ -5,97 +5,91 @@
+            for i in range(0, len(line)):
+                doc[headers[i].lower()] = line[i]
 
             data.append(doc)
-
+            
     return headers, data
 
-
-## Insert the data
+# Insert the data
 def insert_data(headers, data):
     ## Connect to Scylla cluster and create schema
-    # session = cassandra.cluster.Cluster(SCYLLA_IP).connect()
     print("")
     print("## Connecting to Scylla cluster -> Creating schema")
     session = Cluster(SCYLLA_IP).connect()
     session.execute(create_ks)
     session.execute(create_t1)
-
+    
     ## Connect to Elasticsearch
-    print ("")
-    print ("## Connecting to Elasticsearch -> Creating 'Catalog' index")
-    es = Elasticsearch(ES_IP)
+    print("")
+    print("## Connecting to Elasticsearch -> Creating 'Catalog' index")
+    
+    # Ignore status 400 = 'IF NOT EXISTS' for index creation
+    es = Elasticsearch(ES_IP).options(ignore_status=400)
+    
+    ## Create Elasticsearch index
+    es.indices.create(index='catalog')
     
-    ## Create Elasticsearch index. Ignore 400 = IF NOT EXIST
-    es.indices.create(index="catalog", ignore=400)
-
     ## Non-prepared CQL statement
     #cql = "INSERT INTO catalog.apparel(sku,brand,group,sub_group,color,size,gender) VALUES(%(sku)s,%(brand)s,%(group)s,%(sub_group)s,%(color)s,%(size)s,%(gender)s)"
 
     ## Prepared CQL statement
     print("")
-    print("## Preparing CQL statement")
-    cql = "INSERT INTO catalog.apparel (sku,brand,group,sub_group,color,size,gender) VALUES (?,?,?,?,?,?,?) using TIMESTAMP ?"
+    print("## Preparing CQL and ES queries")
+    cql = "INSERT INTO catalog.apparel (sku, brand, group, sub_group, color, size, gender) VALUES (?, ?, ?, ?, ?, ?, ?) USING TIMESTAMP ?"
     cql_prepared = session.prepare(cql)
     cql_prepared.consistency_level = ConsistencyLevel.ONE if random.random() < 0.2 else ConsistencyLevel.QUORUM
 
     print("")
-    print("## Insert csv content into Scylla and Elasticsearch")
-
-
-    for d in data: 
+    print("## Inserting data into Scylla and Elasticsearch")
+    
+    for row in data:
         # See if we need to add code to wait for the ack. This should be synchronous.
         # Also, might need to switch to prepared statements to set the consistency level for sync requests.
-        session.execute(cql_prepared, d)
-
-        res = es.index(index="catalog", doc_type="apparel", id=d["sku"], body=d)
+        session.execute(cql_prepared, row)
+        
+        es.index(index="catalog", id=row["sku"], body=row)
 
-    ## After all the inserts, make a refresh, just in case
+        print(f"Inserted {row['sku']} into Scylla and Elasticsearch")
+    
     print("")
     print("## Inserts completed, refreshing index")
     es.indices.refresh(index="catalog")
-
+    
     print("")
-
-
+        
 if __name__ == "__main__":
-    headers, data = load_data("./catalog.csv")
-    insert_data(headers, data)
+    headers, data = load_data('./catalog.csv')
```

Comment:
Inconsistence between single and double quotes, which may be fixed with the formatter.